virtual/README.md

```diff
@@ -102,8 +97,7 @@ $ lspci -nnk | grep NVIDIA
 
```

Comment:
I thought the `./cluster_up.sh` instruction was confusing and you were going to add an explanation for what it does? Line 54.

Reply:
remind me here, `./cluster_up.sh` deploys the k8s cluster within the vagrant vms created in the `./vagrant_startup.sh` step. correct?

Reply:
Yes. `./vagrant_startup.sh` starts the VMs. `./cluster_up.sh` runs the DeepOps scripts. Specifically, it deploys Kubernetes and/or Slurm.